### ISSUE TITLE
User Screen: Cannot find a user after they have created an account

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/VRMS#1531]
### Overview
As the Org admin, I need to be able to add a user to their project as a pm, so that they can edit meeting info.

### Action Items
- [x] Review screenshots
- [x] Identify issue
- [x] talk with tech lead
- [x] etc.
- [x] once fixed, let the guides team know in the Guides slack channel.

### Resources/Instructions
<details><summary>Screenshot of user management page</summary>

![image](https://github.com/hackforla/VRMS/assets/37763229/8b7faa45-0a58-49ee-b828-22d03d9c8da4)

</details>

**Comment by JackHaeg** [comment]
Based on the screenshots, it appears the poster of this issue searched for a new user's email address while using the "results by name" feature instead of the correct "results by email" feature. This is the cause for the user's account not displaying in the search results. 

As can be seen in the screenshot below, when using the "results by email" feature, the user's account shows up properly. 

<img width="824" alt="Screenshot 2023-10-20 at 11 19 46 AM" src="https://github.com/hackforla/VRMS/assets/134463646/21dedaca-f23f-4455-b17f-056db27497df">

I have added the user to the Guides project on VRMS and notified the user on Slack. Closing issue as complete
